packages/manager/modules/pci/src/projects/new/description/description.html

```diff
@@ -0,0 +1,45 @@
+<form class="pci-projects-new-description"
+      name="projectDescriptionForm">
+
+    <oui-field
+        data-label="{{ :: 'pci_project_new_description_project_name' | translate }}">
+        <input
+            class="oui-input"
+            type="text"
+            id="newProjectName"
+            name="newProjectName"
+            data-ng-model="$ctrl.model.name"
+            maxlength="255">
+    </oui-field>
+
+    <oui-checkbox data-model="$ctrl.model.agreements"
+                  data-name="agreements"
+                  data-required>
+        <span data-translate="pci_project_new_description_project_contracts"></span>
+    </oui-checkbox>
+
+    <div class="d-flex flex-row-reverse justify-content-between align-items-center">
```

Comment:
Is [`form-actions`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/) could fit your needs?